python/cugraph/cugraph/structure/symmetrize.py

```diff
@@ -164,29 +158,14 @@ def symmetrize_ddf(
         weight_name = [weight_name]
 
     if symmetrize:
-        if weight_name:
-            ddf2 = ddf[[*dst_name, *src_name, *weight_name]]
-            ddf2.columns = [*src_name, *dst_name, *weight_name]
-        else:
-            ddf2 = ddf[[*dst_name, *src_name]]
-            ddf2.columns = [*src_name, *dst_name]
-        result = dask_cudf.concat([ddf, ddf2]).reset_index(drop=True)
+        result = ddf.map_partitions(_add_reverse_edges, src_name, dst_name, weight_name)
```

Comment:
Nice, this is a very clever solution!  👍

Reply:
Thanks